c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,54 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
+  var html = '';
+  var header = '<span class="suggestion-text">' +
+                            '<span class="suggestion-title">' +
+                              '<span ng-bind-html="highlight(label, query)"></span>' +
+                              '<span ng-if="date_end">&nbsp;({{::date_end }})</span>' +
+                            '</span><br>' +
+                            '<span class="suggestion-info">' +
+                            '  <span ng-if="elevation">{{::elevation}}m</span>' +
+                            '</span>';
+  var footer =   '</span>';
```

Comment:
you are right, but checking every document type and having a different html for each will multiply the code.
Maybe a less "subtle" solution would be to have the same big template for every doc, but check with ng-ifs for the values in order to show them or not, as it's done now with date which does not appear in other docs than outings.
One way or another we will have to check for values and/or types of document.